src/hooks/useFilterBar.stories.mdx

```diff
@@ -0,0 +1,271 @@
+| Added        | `1.4.0`    |
+| Latest       | `1.4.0`    |
+
+## Usage
+
+Usage of useFilterBar
+
+```tsx
+const {
+  onChange,
+  onMixedStateChange,
+  onClear,
+  onClearAll,
+  setSelectedItems,
+  selectedItems,
+  onOpen,
+  onClose,
+  onToggle,
+  isModalOpen,
+} = useFilterBar();
+```
+
+## useFilterBar Hook simple implementation with no buttons or modul
+
+```jsx
+import React from "react";
+import {
+  FilterBar,
+  MultiSelectGroup,
+  MultiSelect,
+  useFilterBar,
+} from "@nypl/design-system-react-components";
+
+export default function FilterBarWithHookExample() {
+  // Without this hook, you'd have to add handlers for onChange, onMixedStateChange and onClear,
+  // and also manage the selectedItems state yourself.
+  const { onChange, onMixedStateChange, onClear, selectedItems } =
+    useFilterBar();
+  return (
+    <FilterBar
+      id="filterbar"
+      headingText="FilterBar"
+      layout="row"
+      selectedItems={selectedItems}
+    >
+      <MultiSelectGroup
+        id="multiselect-group"
+        key="multiselect-group-key"
+        labelText="MultiSelect Group"
+      >
+        {multiSelectItems &&
+          multiSelectItems.map((multiSelect) => (
+            <MultiSelect
+              key={multiSelect.id}
+              id={multiSelect.id}
+              label={multiSelect.name}
+              variant="dialog"
+              items={multiSelect.items}
+              selectedItems={selectedItems}
+              onChange={(e) => {
+                onChange(e.target.id, multiSelect.id);
+              }}
+              onMixedStateChange={(e) => {
+                onMixedStateChange(
+                  e.target.id,
+                  multiSelect.id,
+                  multiSelect.items
+                );
+              }}
+              onClear={() => {
+                onClear(multiSelect.id);
+              }}
+              onApply={() => {}}
+            />
+          ))}
+      </MultiSelectGroup>
+    </FilterBar>
+  );
+}
+```
+
+## useFilterBar Hook implementation with `Clear All` and `Apply Filters` button
+
+```jsx
+import React from "react";
+import {
+  FilterBar,
+  MultiSelectGroup,
+  MultiSelect,
+  useFilterBar,
+} from "@nypl/design-system-react-components";
+
+export default function FilterBarWithHookExample() {
+  // Without this hook, you'd have to add handlers for onChange, onMixedStateChange, onClear and onClearAll,
+  // and also manage the selectedItems state yourself.
+  const { onChange, onMixedStateChange, onClear, onClearAll, selectedItems } =
+    useFilterBar();
+
+  return (
+    <FilterBar
+      id="filterbar"
+      headingText="FilterBar"
+      layout="row"
+      selectedItems={selectedItems}
+      showClearAll={true}
+      onClear={onClearAll}
+      showSubmitAll={true}
+      onSubmit={() => {}}
+    >
+      <MultiSelectGroup
+        id="multiselect-group"
+        key="multiselect-group-key"
+        labelText="MultiSelect Group"
+      >
+        {multiSelectItems &&
```

Comment:
I know this is an example, but it would be good to define what `multiSelectItems` is as part of the example.